packages/fscomponents/src/components/CartCount.tsx

```diff
@@ -80,16 +82,23 @@ export const CartCount: FunctionComponent<CartCountProps> = memo((props): JSX.El
   const textPosition = props.textPosition || 'center';
 
   return (
-    <View style={[styles.container, props.style]}>
+    <View
+      style={[styles.container, props.style]}
+      accessibilityLabel={
+        FSI18n.number(props.count) + ' ' + FSI18n.string(componentTranslationKeys)
+      }
+    >
       <Image
         resizeMode='contain'
         style={[styles.image, props.cartImageStyle]}
         source={props.cartImage || DEFAULT_CART_IMAGE}
+        accessibilityRole={'image'}
+        accessibilityLabel={
+          FSI18n.number(props.count) + ' ' + FSI18n.string(componentTranslationKeys)
```

Comment:
You construct this same string a few lines up, so it would be better to store it in a variable and then reuse it.